package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@justacodename/sanity-document-internationalization",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

I would not make version changes in PRs, this is something that usually only happens on main (or v1 in this case) just before packaging.
```suggestion
  "version": "1.1.2",
```